smtp/help.md

```diff
@@ -82,6 +82,7 @@ If you are passing a variable into the Attachment 'Content' field, you must make
 
 ## Version History
 
+* 2.0.5 - Fix issue sending emails without attachment
```

Comment:
This becomes a major version bump since you changed the input types on the connection.